MParT/MapFactory.h

```diff
@@ -4,6 +4,8 @@
 #include "MParT/MapOptions.h"
 
 #include "MParT/ConditionalMapBase.h"
+#include "MParT/SummarizedMap.h"
+//#include "MParT/DebugMap.h"
```

Comment:
We'll need to make an issue introducing a "debug build option" of MParT so this will auto build when we make a debug build, but won't otherwise. I'll put it up as an issue after this.